js/NavMainComponent.js

```diff
@@ -74,8 +112,33 @@ NavMainComponent.prototype.switchSecondaryNav = function(target) {
     component.closeSubNavs();
 
     component.$html.find('.nav-list.is-active').removeClass('is-active');
-    component.$html.find('[data-nav="' + target + '"]')
-        .addClass('is-active');
+    component.$html.find('[data-nav="' + target + '"]').addClass('is-active');
+};
+
+NavMainComponent.prototype.switchTertiartyNav = function(target) {
+
+    var component = this;
+
+    component.$html.find('.navTertiary .nav-link').removeClass('is-active');
+
+    if (component.$html.find('[data-nav="' + target + '"]').length < 1) {
+        component.closeNavs();
+    }
+
+    component.$html.find('.navTertiary .is-active').removeClass('is-active');
+    component.$html.find('[href="' + target + '"]').addClass('is-active');
+};
+
+NavMainComponent.prototype.switchQuaternaryNav = function(target) {
+
+    var component = this;
+
+    // If category item has encapsulated options and if it is same-page link then open the proper menu
+    if (target.indexOf('#') > -1) {
```

Comment:
looks like a duplicate method of detecting an ID (duplicate seen on line 62 using `substring`)

I think `indexOf` is slightly more performant (https://jsperf.com/indexof-vs-substring-end-of-string [not perfect test for our use case]) so might be worth changing line 62 to use this method to keep things consistent.